### DISCUSSION TITLE
Tool Used for documenting flutter code?

### DISCUSSION THREAD

**Discussion by mithilesh-parmar** [AppFlowy-IO/AppFlowy#10] [General]
The comment showing the app startup in **startup.dart** caught my attention, I'd like to know the tool (if any) you are using for such documentation.  For reference I'm talking about the below comment

// [[diagram: flowy startup flow]]
//                   ┌──────────┐
//                   │ FlowyApp │
//                   └──────────┘
//                         │  impl
//                         ▼
// ┌────────┐  1.run ┌──────────┐
// │ System │───┬───▶│EntryPoint│
// └────────┘   │    └──────────┘         ┌─────────────────┐
//              │                    ┌──▶ │ RustSDKInitTask │
//              │    ┌───────────┐   │    └─────────────────┘
//              └──▶ │AppLauncher│───┤
//        2.launch   └───────────┘   │    ┌─────────────┐         ┌──────────────────┐      ┌───────────────┐
//                                   └───▶│AppWidgetTask│────────▶│ApplicationWidget │─────▶│ SplashScreen  │
//                                        └─────────────┘         └──────────────────┘      └───────────────┘
//
//                                                 3.build MeterialApp

**Comment by appflowy** [comment]
Glad to hear that. The tool I'm using is the monodraw. Check this link https://monodraw.helftone.com/.
